packages/lab/src/stepped-tracker/TrackerStep/TrackerStep.css

```diff
@@ -0,0 +1,50 @@
+.saltTrackerStep {
+  --trackerStep-icon-color-active: var(--saltTrackerStep-icon-color-active, var(--salt-status-info-foreground));
+  --trackerStep-icon-color-completed: var(--saltTrackerStep-icon-color-completed, var(--salt-status-success-foreground));
+
+  --trackerStep-icon-color: var(--saltTrackerStep-icon-color, var(--salt-status-static-foreground));
+
+  --saltIcon-color: var(--trackerStep-icon-color);
+  /* We redefine Icon Size so we can use it in calc functions in the trackerConnector */
+  --saltIcon-size: var(--saltTrackerStep-icon-size, max(var(--salt-icon-size-base), 12px));
+}
+
+.saltTrackerStep {
+  margin: 0;
+  padding: 0;
+  text-indent: 0;
+  list-style-type: none;
+  flex: 1;
+  position: relative;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: var(--trackerStep-width);
```

Comment:
should we put a fallback for this one in case undefined? I also think that when we inject it from the component styles we usually use the API token --saltTrackerStep-width